src/ServiceProvider.php

```diff
@@ -3,7 +3,8 @@
 namespace SoapBox\Settings;
 
 use SoapBox\Settings\Repositories\Settings;
-use Symfony\Component\Cache\Simple\ArrayCache;
+use Symfony\Component\Cache\Adapter\ArrayAdapter;
+use Symfony\Component\Cache\Psr16Cache;
 use SoapBox\Settings\Repositories\CacheSettings;
 use SoapBox\Settings\Repositories\DatabaseSettings;
 use Illuminate\Support\ServiceProvider as IlluminateServiceProvider;
```

Comment:
order of use statements should be shortest to longest

Reply:
Fixed on commit https://github.com/Soapbox/GoodTalk-Settings/pull/14/commits/1fe0a49130e6e4b71cd8d6e00ccf393a659ffa1f